tardis/__init__.py

```diff
@@ -23,4 +23,3 @@
 # ----------------------------------------------------------------------------
 
 from tardis.base import run_tardis
-from tardis.io.util import yaml_load_config_file as yaml_load
```

Comment:
For backwards compatibility, please change `yaml_load_config_file` to `yaml_load_file` and keep this line. That way if somebody imports `yaml_load` it will still work.